lib/std/winapis.nim

```diff
@@ -0,0 +1,14 @@
+## An inplacement for `std/winlean` with presice and stable types definitions.
+
+type
+  DWORD* = culong  ## https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#ulong
+  ULONGLONG* = int64 ## https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#ulonglong
+  PULONGLONG* = ptr ULONGLONG ## https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#pulong
+
```

Comment:
i was thinking we may want to split in 2:
`std/wintypes` contains type definitions only (DWORD etc); it's a small, finite module (not a heavy dependency)

and `std/winapis` contains, well, APIs (it can grow large)

Reply:
I'm not sure, so I make this PR to let people discuss.

Reply:
maybe plaster the usual caveat on top of the module (experimental API, subject to change) so we can move forward, and decide later if this needs splitting?